### ISSUE TITLE
[@astrojs/image]: Problems with <Picture /> and scrolling

### ISSUE THREAD

**Issue by panwauu** [withastro/astro#5010]
### What version of `astro` are you using?

1.4.4

### Are you using an SSR adapter? If so, which one?

None

### What package manager are you using?

npm

### What operating system are you using?

Windows

### Describe the Bug

I noticed some weird problems with the <Picture /> component and scroll behavior. I added 20 pictures and you can see that scroll is not correct.

- Scrolling with mouse seems normal, but the scrollbar is flickering and the side is changing
- If you try to click and drag the scrollbar to scroll, the cursor changes position relative to the scrollbar and it is not usable.

I tried adding picture elements manually and the scroll behavior is normal.
Glad for any help.

### Link to Minimal Reproducible Example

https://stackblitz.com/edit/github-qnk2xr-vxtnoe?file=src%2Fpages%2Findex.astro&on=stackblitz

### Participation

- [X] I am willing to submit a pull request for this issue.

**Comment by matthewp** [comment]
What is the proposed bug here? `@astrojs/image` does not affect the scrollbar. If the scrollbar is flickering it is likely a bug with your browser. The image component just creates a `<picture>` element. Perhaps some combination of the attributes added causes this.

If you figure out what the attributes are and they are something that we can avoid using, please open a new issue.

**Comment by panwauu** [comment]
I did not figure out what attributes are the problem, but in the stackblitz I included an example with custom picture elements instead of the <Picture /> component and the scrolling problem does not appear with these.

I did not figure out how to fix it, but it keeps me from using <Picture /> currently, so I think this is a valid Issue.

**Comment by matthewp** [comment]
The HTML snippet doesn't include the same attributes as what Picture produces. Create an example with all of the same attributes and I'm guessing you will get the same problem.

**Comment by panwauu** [comment]
I reworked the example by creating another page `working` with the same output as the image component only without the classes applied by the <Picture /> component. And I found the problem:
```css
img:where(.astro-EI35XRNH) {
    content-visibility: auto;
}
```
If I set the property to `visible` there is no problem with the scrolling. 

BTW the problem was not visible in firefox, which does also not support the `content-visibility` property. I could not test safari, which does also not support it. See [caniuse](https://caniuse.com/?search=content-visibility)

Still see:
https://stackblitz.com/edit/github-qnk2xr-vxtnoe?file=src%2Fpages%2Findex.astro&on=stackblitz
